tests/unit/tuf_repository_service_worker/test_repository.py

```diff
@@ -481,7 +502,12 @@ def test_update_snapshot_bump_all(self, test_repo, monkeypatch):
         targets = ["bins-e", "bins-f"]
         result = test_repo._update_snapshot(targets, bump_all=True)
 
-        assert result == snapshot_version
+        assert result == 4
+        assert mocked_snapshot.signed.version == 4
```

Comment:
See my other comment regarding this check.
```suggestion
        assert result == snapshot_version + 1
        assert mocked_snapshot.signed.version == snapshot_version + 1
```